src/components/Psychologist/Patients/Patients.jsx

```diff
@@ -65,19 +61,18 @@ function Patients() {
 								<p>No tienes pacientes registrados.</p>
 							</div>
 						)}
+					<Link
+						to="/dashboard/AddPatients/"
+					>
+						<input
+							className={styles.btnAdd}
+							type="submit"
+							value="Agregar"
+						/>
+					</Link>
```

Comment:
Remove the button
Leave only the link if you are going to change the route and style the link as a button